lib/vfscore/mount.c

```diff
@@ -377,12 +377,14 @@ UK_LLSYSCALL_R_DEFINE(int, sync)
 	bio_sync();
 #endif
 	uk_mutex_unlock(&mount_lock);
+
+	return 0;
 }
 
 #if UK_LIBC_SYSCALLS
 void sync(void)
 {
-   uk_syscall_e_sync();
+	uk_syscall_e_sync();
```

Comment:
Hi @skuenzer, thanks for sorting this fix! It all looks good, but can you please remove this indentation change in `sync` from commit bab546d.

Reply:
Hey, why :thinking:? Indentation should be tabs, this was originally wrong. This one liner is also such a minimal change so that I think it is okay to have it as part of the `sync` fix, especially since it is related. 😄

Reply:
Agree with @skuenzer. Don't think this change would justify a dedicated PR, but it should be corrected. So this seems to be a good opportunity, although I understand @nderjung that it is not strictly part of the originally address problem.

Reply:
I am very [keen on fixing stylistic inconsistencies](https://github.com/unikraft/unikraft/issues/199) 😀 However, in my opinion, the approach should not be to simultaneously fix a separate stylistic error in a file where the commit occurs, as this undocumented and separate change will disrupt `git-blame`.  Unless the commit message also indicates the change or the main diff happens to touch that line and fix the style, then i think it should be a separate commit entirely. 😄

Reply:
We should probably start deciding and document how we want to handle minor style fixes within a related patch. As compromise, I added a comment in the patch description about this style fix.